%YAML:1.0
---
left_image_width: 752
left_image_height: 480
left_camera_name: "camera/left"
left_camera_matrix: !!opencv-matrix
  rows: 3
  cols: 3
  dt: d
  data: [458.654, 0.000000, 367.215, 0.000000, 457.296,248.375, 0.000000, 0.000000, 1.000000]
left_distortion_model: radial-tangential
left_distortion_coefficients: !!opencv-matrix
  rows: 1
  cols: 5
  dt: d
  data: [-0.28340811, 0.07395907,0.00, 0.00019359, 1.76187114e-05]
# Sensor extrinsics wrt. the body-frame.
left_T_BS: !!opencv-matrix
  cols: 4
  rows: 4
  dt: d
  data: [0.0148655429818, -0.999880929698, 0.00414029679422, -0.0216401454975,
         0.999557249008, 0.0149672133247, 0.025715529948, -0.064676986768,
        -0.0257744366974, 0.00375618835797, 0.999660727178, 0.00981073058949,
         0.0, 0.0, 0.0, 1.0]

right_image_width: 752
right_image_height: 480
right_camera_name: "camera/right"
right_camera_matrix: !!opencv-matrix
  rows: 3
  cols: 3
  dt: d
  data: [457.587, 0.000000, 379.999, 0.000000, 456.134,255.238, 0.000000, 0.000000, 1.000000]
right_distortion_model: radial-tangential
right_distortion_coefficients: !!opencv-matrix
  rows: 1
  cols: 5
  dt: d
  data: [-0.28368365,  0.07451284, 0.00, -0.00010473, -3.55590700e-05]
# Sensor extrinsics wrt. the body-frame.
right_T_BS: !!opencv-matrix
  cols: 4
  rows: 4
  dt: d
  data: [0.0125552670891, -0.999755099723, 0.0182237714554, -0.0198435579556,
         0.999598781151, 0.0130119051815, 0.0251588363115, 0.0453689425024,
        -0.0253898008918, 0.0179005838253, 0.999517347078, 0.00786212447038,
         0.0, 0.0, 0.0, 1.0]
